report/finished/chapters/motion.tex

```diff
@@ -1,4 +1,40 @@
-\section*{Robot arm}
+\section*{Robot Arm Path Planning} 
+
+
+
+If a robot manipulator is required to visit specific discrete points, waypoints, path planning is the problem of deciding how to move between these points. Planning the path is also important to ensure the movements are smooth, meaning that, since we "control" the speed of each joint, the desired positions and velocities should be continuous.
+%Path planning is very important, a smooth path will make a manipulator move better :) %is right
+\subsection*{Interpolation}
+One method of interpolating between points is to use cubic spline interpolation. A spline \cite{spline_wolfram} is a piecewise polynomial function and consequently a cubic spline is a function consisting of piecewise cubic polynomials. A cubic polynomial will ensure a smooth movement since its first derivative, the velocity, will be continuous. To ensure that each required point is visited the bounding values for each cubic function are set to zero \cite{cubic_spline_wolfram}.
+
+\subsection*{Joint Space interpolation or Cartesian Motion}
+The interpolation can take place in joint space or in cartesian space. Joint space interpolation means that the waypoints are expressed as joint values and that the interpolation is calculated between these joint values. Cartesian motion means that the waypoint are expressed as cartesian coordinates and that the interpolation takes place in cartesian space.
```

Comment:
mention that joints are angles

Reply:
Good point, joint values can be either angles or distance. I will add it